scripts/init.sh

```diff
@@ -23,9 +42,11 @@ done
 
 # Configure rp_filter
 echo "Configure rp_filter loose... "
+
 TOKEN=$(curl -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 60")
```

Comment:
Nit: Could you move this command into `get_metadata()` function as well ?

Reply:
I had it initially inside get_metadata but that will lead to getting a token for each call. Hence moved it outside. Do you see any issue here?

Reply:
Nope :D